contents/blog/best-hotjar-alternatives.md

```diff
@@ -0,0 +1,431 @@
+---
+title: The best Hotjar alternatives & competitors, compared
+date: 2024-03-04
+author: ["andy-vandervell"]
+rootpage: /blog
+featuredImage: ../images/blog/hotjar-alternatives/hotjar-alternatives.png
+featuredImageType: full
+category: General
+tags:
+ - Comparisons
+---
+
+import { ComparisonTable } from 'components/ComparisonTable'
+import { ComparisonRow } from 'components/ComparisonTable/row'
+
+## 1. PostHog
```

Comment:
Sometimes I feel we should do something like this so that it makes it obvious that its us and feels less like us doing "sneaky shit"

feel free to ignore

```suggestion
## 1. PostHog (that's us 👋🏼)
```

Reply:
Oh I see we have further below